README.md

```diff
@@ -1,2 +1,177 @@
+### Key Features <a name="key-features"></a>
+
+
+- Add index.html file.
+- Add style.css file.
+- Add index.js file.
+- Add Linters.
+- Add Header to the index.html file.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+1-Web Browser
+2-Code Editor
+3-Git & Github
+4-Node.js
+5- WebPack.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Run this Command:
+
+```sh
+  cd Leaderboard App
+  git clone git@github.com:AbuTalha3/Leaderboard-app.git
+```
+
+### Install
+
+First you need to install the node package files
+Run this command:
+
+```sh
+  cd Leaderboard App
+  npm install
+```
+<!-- START APP -->
+### Start app
+While using app for the first time, Use VS code live server Extensions.
+
+### Usage
+
+- In order to use this project you either create branch to add more features.
+- But in order to run the project open index.html with live server
+
+### Run Linters
+
+I have added the linters or tests in order to avoid anykind of typos and ensure the good practices in this project. So while using this repository if you need to check your work in html and css you can use below commands
+
+- in order to run the linter test on index.html:
+
+```sh
+ npx hint .
+```
+
+- in order to run the linter test on stylesheets:
+
+```sh
+ npx styleint "**/*.{css,scss}"
+```
+
+### Deployment
+
+- You can easily use github pages, netlify or vercel to deploy your project
+  <a href="https://docs.github.com/en/pages/quickstart">Useful link about Github Pages</a>>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 Abu Talha
+
+- GitHub: [@githubhandle](https://github.com/AbuTalha3)
+- Twitter: [@twitterhandle](https://twitter.com/AbuTalha8T)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/abu-talha-8203b252/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] Add header and footer to index.html
+- [ ] Add more styles to style.css
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/AbuTalha3/Leaderboard-app/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project just give it a star -
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
Please ensure to link this link properly to your license file as it is currently routing to a 404 page which is making the readme a little less professional